app/views/steps/income/client/employment_details/edit.html.erb

```diff
@@ -7,14 +7,12 @@
     <%= govuk_error_summary(@form_object) %>
 
     <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
-
+    <% @form_object.before_or_after_tax = @form_object.record.before_or_after_tax['value'] if @form_object.record.before_or_after_tax %>
```

Comment:
Personally I'd prefer to have this logic in the Form class rather than the view but probably just a preference thing!

Reply:
👍 Agree. Even i am liking this solution. Will fix it with another PR